src/app.tsx

```diff
@@ -183,63 +197,56 @@ export const Application = () => {
-                              variant={alert.variant}
-                              title={alert.title}
-                              actionClose={
-                                  <AlertActionCloseButton
-                                    title={alert.title}
-                                    variantLabel={`${alert.variant} alert`}
-                                    onClose={() => removeAlert(alert.key)}
-                                  />
-                              }
-                              actionLinks={alert.actionLinks}
-                              key={alert.key}
-                            >
-                                {alert.detail}
-                            </Alert>
-                        ))}
-                    </AlertGroup>
-                    <FilesBreadcrumbs path={path} />
-                    <PageSection
-                      hasBodyWrapper={false}
-                      id="files-folder-section"
-                      data-dir-loaded={!loadingFiles ? path : null}
-                    >
+                    <Page className="pf-m-no-sidebar" isContentFilled>
+                        <AlertGroup isToast isLiveRegion>
+                            {alerts.map(alert => (
+                                <Alert
+                                  variant={alert.variant}
+                                  title={alert.title}
+                                  actionClose={
+                                      <AlertActionCloseButton
+                                        title={alert.title}
+                                        variantLabel={`${alert.variant} alert`}
+                                        onClose={() => removeAlert(alert.key)}
+                                      />
+                                  }
+                                  actionLinks={alert.actionLinks}
+                                  key={alert.key}
+                                >
+                                    {alert.detail}
+                                </Alert>
+                            ))}
+                        </AlertGroup>
+                        <FilesBreadcrumbs path={path} />
                         {errorMessage &&
-                        <Card className="files-empty-state">
-                            <EmptyStatePanel
-                              paragraph={errorMessage}
-                              icon={ExclamationCircleIcon}
-                            />
-                        </Card>}
+                        <PageSection
+                          hasBodyWrapper={false}
+                        >
+                            <Card className="files-empty-state">
+                                <EmptyStatePanel
+                                  paragraph={errorMessage}
+                                  icon={ExclamationCircleIcon}
+                                />
+                            </Card>
+                        </PageSection>}
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-1237-6887ed2d-20251114-141848-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)